packages/api/src/controllers/user.ts

```diff
@@ -409,6 +413,16 @@ app.post("/", validatePost("user"), async (req, res) => {
     return res.json({ errors: ["user not created"] });
   }
 
+  if (
+    req.config.requireEmailVerification &&
+    !user.emailValid &&
+    !user.admin &&
+    !req.config.isTest
+  ) {
+    res.status(403);
+    return res.json({ errors: ["we just sent you a verification email"] });
+  }
```

Comment:
I don't think this check makes sense. Wasn't this user just created with `emailValid = validUser` var? Either verification will be disabled and this always passes, or verification is enabled and the registration request always fails. (this is the registration API right?)

Reply:
yeah - it's failing a returning the text to be displayed under the form - similar to "email already registered" appearing just over the sign up button

I agree this should'nt be a 403 and a better way to display messages should exist

Reply:
Definitely should not be a 403.